### PR TITLE
Update check_updates.py

### PR DESCRIPTION
**Pull request by XPOHUK**
Set colour_have_updates only when number of updates > 0
Now this color is set even if the number of updates is 0

### CODE REVIEW

**Comment by ramnes** [comment]
This won't work if we enter the condition in line 78.

**Comment by flacjacket** [comment]
It looks like that is actually the correct behavior, Ramnes, the addition in that block is actually string concatenation.

**Comment by ramnes** [comment]
@flacjacket Because it is a string concatenation, it can't be cast as int.

File: libqtile/widget/check_updates.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def _check_updates(self):
 
     def _set_colour(self, num_updates):
         # type: (str) -> None
-        if num_updates:
+        if int(num_updates) > 0:
```
Comment (ramnes) [inline libqtile/widget/check_updates.py]: hackish but works:

```suggestion
        if not num_updates.startswith("0"):
```

**Comment by ramnes** [comment]
Thanks!

**Comment by XPOHUK** [comment]
Not at all!

**Pull request merged**